ui/app/src/models/Plugin.ts

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
Add "Marketplace" to names, including file name

Reply:
will create another PR for this